.travis.yml

```diff
@@ -46,6 +46,9 @@ matrix:
       env:
         - MB_PYTHON_VERSION=3.7
         - PLAT=i686
+    - os: linux
+      env:
+        - MB_PYTHON_VERSION=3.8
```

Comment:
Could you try here to use the manylinux2010 image by adding `MB_ML_VER=2010`? In order to use this you will also need to update the multibuild subrepo to the HEAD of the devel branch

Reply:
Sadly, it failed at the same spot, it cannot find `pip`.

Reply:
Did download 2010 however.

Reply:
The path in the docker image is `/opt/python/cp38-cp38`, multibuild expects a `m` suffix `/opt/python/cp38-cp38m`

Reply:
I could test locally on ubuntu 18.04 with
```
##!bash
set -x
REPO_DIR=numpy
        # Also see DAILY_COMMIT below
BUILD_COMMIT=master
BUILD_DEPENDS=cython
TEST_DEPENDS=pytest
PLAT=x86_64
UNICODE_WIDTH=32
WHEELHOUSE_UPLOADER_USERNAME=travis-worker
MB_PYTHON_VERSION=3.8
MB_ML_VER=2010
source multibuild/common_utils.sh
source multibuild/travis_steps.sh
before_install
clean_code $REPO_DIR $BUILD_COMMIT
./patch_code.sh $REPO_DIR
if [ "$TRAVIS_OS_NAME"  == "linux" ]; then
    export CFLAGS=${CFLAGS}" -Wno-sign-compare -Wno-unused-result\
                             -Wno-strict-aliasing";
fi
build_wheel $REPO_DIR $PLAT
```

and to debug the docker I added `set -x` to the top of `multibuild/docker_build_wrap.sh`

Reply:
What is the difference between a python with and without the `m` suffix?